datadome-dotnetcore-demo/datadome-dotnetcore-demo.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <RootNamespace>datadome_dotnetcore_demo</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="DataDome.AspNetCore" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="DataDome.AspNetCore" Version="1.0.0" />
```

Comment:
So this is where you can specify the version of the module, right ?

Reply:
yes @MickaelDatadome 
the version of the nuget package: https://www.nuget.org/packages/DataDome.AspNetCore